requirements/base.txt

```diff
@@ -57,9 +57,9 @@ newrelic==7.2.4.171
     # via edx-django-utils
 pbr==5.8.0
     # via stevedore
-pillow==8.4.0
+pillow==9.0.0
```

Comment:
There isn't any breaking change mentioned in the `pillow==9.0.0` [CHANGELOG]( https://github.com/python-pillow/Pillow/blob/main/CHANGES.rst#900-2022-01-02) so it should be good to merge.